api/pods/pod_handler.go

```diff
@@ -43,12 +45,28 @@ func (ph PodHandler) HandleGetEnvironmentScheduledJobLog(appName, envName, sched
 	return ph.getScheduledJobLog(envNs, scheduledJobName, containerName, sinceTime)
 }
 
+// HandleGetEnvironmentPodLog Get logs from pod in environment
```

Comment:
Method name in comment does not match to real name `HandleGetEnvironmentAuxiliaryResourcePodLog`

Reply:
fixed